app/source/js/viewmodels/AppointmentCard.js

```diff
@@ -256,11 +265,6 @@ function AppointmentCardViewModel(params) {
         return b ? b.canBeCancelledByServiceProfessional() : false;
     }, this);
 
-    this.bookingCanBeCancelledByClient = ko.computed(function() {
```

Comment:
Why remove this? Is used at app\source\js\viewmodels\EditClientBookingCardVM.js because of call to cancelBookingByClient